exercises/beer-song/beer_song_test.py

```diff
@@ -1,70 +1,389 @@
 import unittest
 
-from beer_song import song, verse
+from beer_song import recite
+
+
+# Tests adapted from `problem-specifications//canonical-data.json` @ v2.1.0
+
+SONG = [
```

Comment:
Minor point, but I think it might be better to move this massive chunk of text to after the test class. Normally I would always expect constants to be at the start, but when completing exercises, I regularly look at the test file to see what the tests look like and having to scroll past ~300 lines would be a bit annoying.

Reply:
Good point, I'll move it.